packages/components/src/internal/components/samples/constants.ts

```diff
@@ -315,3 +317,133 @@ export const STORED_AMOUNT_FIELDS = {
 };
 
 export const SAMPLE_TYPE_NAME_EXPRESSION_TOPIC = 'sampleIDs#patterns';
+
+export const DEFAULT_AVAILABLE_STATUS_COLOR = '#F0F8ED';
+export const DEFAULT_CONSUMED_STATUS_COLOR = '#FCF8E3';
+export const DEFAULT_LOCKED_STATUS_COLOR = '#FDE6E6';
```

Comment:
minor: these defaults are used in several places below for SAMPLE_STATUS_COLORS. should we use the constants there?